app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,31 @@
+class CommentsController < ApplicationController
+  before_action :set_post, only: [:create, :destroy]
+
+  def new
+    @comment = Comment.new
```

Comment:
We have a line in `posts_controller.rb` that says `Comment.new`. How different is this one from that?